src/window/mod.rs

```diff
@@ -191,7 +196,7 @@ pub(crate) fn build_windower_builder(
 }
 
 /// Defines the sense of time for a windowing operator.
-pub(crate) trait Clock<V> {
+pub(crate) trait Clock<V: ToPyObject> {
```

Comment:
Uh, didn't realize the repackaging as a pure Rust crate was a possibility, I'm going to be more careful about this in the future. Also, not changing the whole trait for just one kind of clock makes sense, fixed it.
I still need the `build_clock_builder` function require that `V` is a `TdPyAny` for any clock, not sure how to avoid this for now, but it's a more contained requirement at least.